ui/scss/component/_expandable.scss

```diff
@@ -1,14 +1,17 @@
 $COLLAPSED_HEIGHT: 120px;
 
+.expandable {
+  max-height: 120px;
```

Comment:
- `COLLAPSED_HEIGHT` is the only link with the javascript variable, so should use it.
- (optional)Maybe also help add a comment beside `COLLAPSED_HEIGHT` that it needs to match the javascript version.